views/Annotations/AnnotationsLegend.svelte

```diff
@@ -0,0 +1,27 @@
+<script>
+  import AnnotationPoint from "./AnnotationPoint.svelte";
+  
+  // General props
+  export let annotations = [];
+  export let annotationRadius = 8;
+  
+  let x = (annotationRadius + 1);
+  let y = (annotationRadius + 1);
+</script>
+
+<div class="q-choropleth-legend-annotations s-font-note" style="display: flex; flex-direction: column;">
+  {#each annotations as { id, annotation }}
+    <div class="q-choropleth-legend-annotation" style="display: flex; flex-direction: row; margin-top: 4px;">
+      <div style="height: 18px; width: 18px; margin-right: 4px;">
```

Comment:
No inline-styles

Reply:
Tbd as new project in next monthly: https://3.basecamp.com/3500782/buckets/1333472/documents/3339485811